src/runtime/rpc/rpc_endpoint.cc

```diff
@@ -1042,7 +1057,23 @@ class RPCClientSession : public RPCSession, public DeviceAPI {
   bool IsLocalSession() const final { return false; }
 
  private:
+  void RPCMaxTransferRemoteReturnValue(TVMArgs args) {
+    // Use args[1] as return value, args[0] is tcode
+    rpc_chunk_max_size_bytes_ = (int64_t)args[1];
+  }
+
+  void SetRPCMaxTransferSize() {
```

Comment:
might consider renaming to GetRPCMaxTransferSize and/or making this the getter function which handles lazily fetching from server as needed.

Reply:
Done!